.github/workflows/changelog.yml

```diff
@@ -87,12 +85,8 @@ jobs:
       - name: Print changelog preview
         run: cat changelog_preview.md
 
-      - name: Install markdown-link-check
-        run: npm install -g markdown-link-check@${{ env.MARKDOWN_LINK_CHECK_VERSION }}
-
-      - name: Run markdown-link-check
-        run: |
-          markdown-link-check \
-            --verbose \
-            --config .markdown_link_check_config.json \
-            changelog_preview.md
+      - name: Run Lychee
+        uses: lycheeverse/lychee-action@v2.3.0
```

Comment:
what about renovate updates?

Reply:
since this is just a github action, renovate will automatically pick it up. let's see how it goes and if we need to, we can pin down the CLI version and then put that under renovate with usual annotation comment